### ISSUE TITLE
Problem Installing module

### ISSUE THREAD

**Issue by lifeboatpres** [snoack/python-goto#6]
After I installed this module, I was told:

No module named goto

I had this problem on a Centos 6 instance running Python 2.6.6 and a Centos 7 instance running Python 2.7.5.

I solved the problem by changing

`modules=['goto'],`
to
`py_modules=['goto'],`
in setup.py

I was installing with `python setup.py install`

Here is my first attempt to install the module and to run the test script at https://github.com/snoack/python-goto

```
[lifeboat@dev ~]$ cd python-goto-master
[lifeboat@dev python-goto-master]$ sudo python setup.py install
/usr/lib64/python2.7/distutils/dist.py:267: UserWarning: Unknown distribution option: 'modules'
  warnings.warn(msg)
running install
running bdist_egg
running egg_info
creating goto.egg-info
writing goto.egg-info/PKG-INFO
writing top-level names to goto.egg-info/top_level.txt
writing dependency_links to goto.egg-info/dependency_links.txt
writing manifest file 'goto.egg-info/SOURCES.txt'
reading manifest file 'goto.egg-info/SOURCES.txt'
writing manifest file 'goto.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
warning: install_lib: 'build/lib' does not exist -- no Python modules to install

creating build
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying goto.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying goto.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying goto.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying goto.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/goto-0.0.0-py2.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing goto-0.0.0-py2.7.egg
Copying goto-0.0.0-py2.7.egg to /usr/lib/python2.7/site-packages
Adding goto 0.0.0 to easy-install.pth file

Installed /usr/lib/python2.7/site-packages/goto-0.0.0-py2.7.egg
Processing dependencies for goto==0.0.0
Finished processing dependencies for goto==0.0.0
[lifeboat@dev python-goto-master]$ cd
[lifeboat@dev ~]$ python test.py
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from goto import with_goto
ImportError: No module named goto
```

**Comment by snoack** [comment]
Reproduced and fixed: 7bd7540

It indeed should be `py_modules` instead `modules`.

**Comment by Arjunlugade28** [comment]
i have also ImportError: cannot import name 'goto' from 'goto' problem. what i can do to solve that problem.
